dmriprep/interfaces/images.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+"""
+Image tools interfaces
+~~~~~~~~~~~~~~~~~~~~~~
+
+"""
+
+from nipype import logging
+from nipype.interfaces.base import (
+    TraitedSpec, BaseInterfaceInputSpec, SimpleInterface, File
+)
+
+LOGGER = logging.getLogger('nipype.interface')
+
+
+class MeanB0InputSpec(BaseInterfaceInputSpec):
+    in_file = File(exists=True, mandatory=True, desc='dwi file')
+    in_bval = File(exists=True, mandatory=True,
+                   desc='File containing the b-values for all volumes in the dwi series')
+    in_bvec = File(exists=True, mandatory=True,
+                   desc='File containing the b-vectors for all volumes in the dwi series')
+
+
+class MeanB0OutputSpec(TraitedSpec):
+    out_file = File(exists=True, desc='mean b0 file')
+
+
+class MeanB0(SimpleInterface):
+    input_spec = MeanB0InputSpec
+    output_spec = MeanB0OutputSpec
+
+    def _run_interface(self, runtime):
+        self._results['out_file'] = mean_b0(
+            self.inputs.in_file,
+            self.inputs.in_bval,
+            self.inputs.in_bvec,
+            newpath=runtime.cwd)
+        return runtime
+
+
+def mean_b0(in_file, in_bval, in_bvec, newpath=None):
```

Comment:
Can we assume the format proposed in bids-standard/bids-specification#352 for this?

```suggestion
def mean_b0(in_file, in_rasb, b0_thres=50, newpath=None):
```